### PR TITLE
Build instructions for Windows

### PR DESCRIPTION
**Pull request by vsalvino**
Finally writing down the build instructions for Windows (or any other system without `make` installed). I had to use these myself plus I have seen some similar questions come up in the Slack.

Couple other related changes:
* Install the local package as part of requirements.txt
* Ignore all hidden dot directories during flake8.

### CODE REVIEW

File: docs/development.rst [hunk 1]
```diff
@@ -58,15 +82,83 @@ Rebuild and install from Python wheel package::
...
+Make a Python virtual environment. Let's make it in a folder called ``.venv``
+which will be ignored by git.
+
+.. code-block:: shell
+
+   # Create the venv
+   python -m venv ./.venv/
+
+   # Activate it (PowerShell)
+   ./.venv/Scripts/Activate.ps1
```
Comment (dragon-dxw) [inline docs/development.rst:116]: I doubt this command would be identical on Mac or Linux?
(Presumably this'd be `. .venv/bin/activate`?)

**Comment by lb-** [comment]
@vsalvino  can you please rebase this and also review the changes to https://github.com/wagtail/sphinx_wagtail_theme/pull/162/files

There have been some recent changes to requirements files with that PR and the Percy one

File: requirements-dev.txt [hunk 2]
```diff
@@ -1,4 +1,7 @@
--r ./requirements.txt
```
Comment (tbrlpld) [inline requirements-dev.txt:1]: We need this. Please do not remove the installation of the themes dependencies.

**Review by tbrlpld — CHANGES_REQUESTED** [review]
Percy is a tool for generating visual regression tests. The added package helps create DOM snapshot in a live server test with Selenium.

But, that should not affect you. The other changes to the dependency structure might. We do have actual dependencies that are needed to install the theme. We will need those in development too.

File: requirements-dev.txt (hunk 2 above)
Comment (vsalvino) [reply requirements-dev.txt:1]: This is related to the line/comment below (`-e .`) That line will install the package (and consequently its dependencies).

Comment (tbrlpld) [reply requirements-dev.txt:1]: Ah, yea, that makes sense.

File: requirements-dev.txt [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
+# Install package into itself.
+-e .
```
[the remark this answers is not in the document]
Comment (tbrlpld) [reply requirements-dev.txt:2]: Yea, I guess that is fair.

**Review by tbrlpld — APPROVED** [review]
Thanks @vsalvino this looks good to me.

**Review by allcaps — APPROVED** [review]

**Pull request merged**